lektor/buildfailures.py

```diff
@@ -63,7 +64,7 @@ def store_failure(self, artifact_name, exc_info):
             os.makedirs(os.path.dirname(fn))
         except OSError:
             pass
-        with open(fn, 'wb') as f:
+        with open(fn, 'w') as f:
```

Comment:
Please do not open files in text mode on Python 3. This does not do what you think it does. In particular this will not pick the correct encoding.